hoomd/md/pytest/test_methods.py

```diff
@@ -182,46 +177,122 @@ def test_attributes_attached(simulation_factory, two_particle_snapshot_factory,
     check_instance_attrs(method, method_base_params.changed_params, True)
 
 
-def test_rattle_attributes(method_base_params):
-    if not method_base_params.has_rattle:
-        pytest.skip("RATTLE method is not implemented for this method")
+def test_detached(simulation_factory, two_particle_snapshot_factory,
```

Comment:
I think this test could use a more descriptive name.
```suggestion
def test_switch_methods(simulation_factory, two_particle_snapshot_factory,
```